pioperf_rearr tool does not build on Theta

The pioperf_rearr tool (pioperformance_rearr.F90) does not build 
on Theta. A build of master (pio2_0_15-2924-g77266226) fails with
the following error message,
```
cd /home/jayesh/acme/pio/ParallelIO_build/tests/performance && /opt/cray/pe/craype/2.5.12/bin/ftn  -DCPRCRAY -DHAVE_MPI -DINCLUDE_CMAKE_FCI -DLINUX -DPIO_MICRO_TIMING -DTIMING -DTIMING_INTERNAL -DUSE_PNETCDF_VARN -DUSE_PNETCDF_VARN_ON_READ -D_NETCDF -D_NETCDF4 -D_PNETCDF -I/home/jayesh/acme/pio/ParallelIO/src/flib -I/home/jayesh/acme/pio/ParallelIO_build/src/flib -I/opt/cray/pe/netcdf-hdf5parallel/4.4.1.1.3/CRAY/8.6/include -I/opt/cray/pe/hdf5-parallel/1.10.0.3/CRAY/8.6/include -I/usr/include -I/opt/cray/pe/parallel-netcdf/1.8.1.3/CRAY/8.6/include -I/home/jayesh/acme/pio/ParallelIO/src/clib -I/home/jayesh/acme/pio/ParallelIO/src/gptl -I/home/jayesh/acme/pio/ParallelIO_build/src/gptl  -em -J.   -c /home/jayesh/acme/pio/ParallelIO/tests/performance/pioperformance_rearr.F90 -o CMakeFiles/pioperf_rearr.dir/pioperformance_rearr.F90.o
PtrToInt source must be pointer
  %r2893 = ptrtoint void <badref> to i32, !dbg !3153
PtrToInt source must be pointer
  %r3299 = ptrtoint void <badref> to i32, !dbg !3301
PtrToInt source must be pointer
  %r3633 = ptrtoint void <badref> to i32, !dbg !3422
PtrToInt source must be pointer
  %r3826 = ptrtoint void <badref> to i32, !dbg !3489
LLVM ERROR: Broken function found, compilation aborted!
tests/performance/CMakeFiles/pioperf_rearr.dir/build.make:62: recipe for target 'tests/performance/CMakeFiles/pioperf_rearr.dir/pioperformance_rearr.F90.o' failed
make[3]: *** [tests/performance/CMakeFiles/pioperf_rearr.dir/pioperformance_rearr.F90.o] Error 1
make[3]: Leaving directory '/gpfs/mira-home/jayesh/acme/pio/ParallelIO_build'
CMakeFiles/Makefile2:3128: recipe for target 'tests/performance/CMakeFiles/pioperf_rearr.dir/all' failed
make[2]: *** [tests/performance/CMakeFiles/pioperf_rearr.dir/all] Error 2
make[2]: Leaving directory '/gpfs/mira-home/jayesh/acme/pio/ParallelIO_build'
CMakeFiles/Makefile2:544: recipe for target 'CMakeFiles/tests.dir/rule' failed
make[1]: *** [CMakeFiles/tests.dir/rule] Error 2
make[1]: Leaving directory '/gpfs/mira-home/jayesh/acme/pio/ParallelIO_build'
Makefile:342: recipe for target 'tests' failed
make: *** [tests] Error 2

```

--- a/tests/performance/pioperformance_rearr.F90
+++ b/tests/performance/pioperformance_rearr.F90
@@ -561,10 +561,16 @@ contains
   subroutine get_tstamp(wall, sys, usr)
     use perf_mod
     double precision, intent(out) :: wall, sys, usr
-    external :: gptlstamp
+    interface
+      function gptlstamp(wall, sys, usr)
+        integer :: gptlstamp
+        double precision, intent(out) :: wall, sys, usr
+      end function
+    end interface
+    integer :: ierr
 
     if(use_gptl) then
-      call gptlstamp(wall, sys, usr)
+      ierr = gptlstamp(wall, sys, usr)
     else
       call t_stampf(wall, sys, usr)
     end if

